packages/manager/modules/pci/src/projects/project/storages/databases/add/add.html

```diff
@@ -26,289 +26,464 @@
-                data-on-accept="$ctrl.acceptLab(accepted)"
-                data-on-contract-click="$ctrl.clickOnContract()"
-            ></pci-project-lab-agreements>
-        </div>
-        <div data-ng-if="$ctrl.processingOrder">
-            <oui-spinner data-size="s"></oui-spinner>
-            <span
-                data-translate="pci_database_add_create_database_processing"
-            ></span>
+                <h2>{{:: 'pci_database_add_step6_title' | translate }}</h2>
+                <div class="mb-3">
+                    <pci-project-lab-agreements
+                        data-lab="$ctrl.lab"
+                        data-ng-if="!$ctrl.lab.isActivated()"
+                        data-on-accept="$ctrl.acceptLab(accepted)"
+                        data-on-contract-click="$ctrl.clickOnContract()"
+                    ></pci-project-lab-agreements>
+                </div>
+            </div>
+        </form>
+    </div>
+    <div class="col-lg-3">
+        <div class="order-summary sticky-top mb-3">
+            <oui-tile
+                data-heading="{{:: 'pci_database_add_spec_heading' | translate }}"
+            >
+                <oui-tile-definition>
+                    <oui-tile-description>
+                        <div>
+                            <strong
+                                data-translate="pci_database_add_spec_name"
+                            ></strong
+                            ><span
+                                ng-bind="$ctrl.model.name"
+                                class="ml-2"
+                                oui-popover="lorem ipsum"
+                            ></span>
+                            <button
+                                type="button"
+                                class="oui-popover-button"
+                                oui-popover="{{:: 'pci_database_add_spec_name_popover' | translate }}"
+                                oui-popover-id="popover-name"
+                                oui-popover-placement="right"
+                            ></button>
+                        </div>
+                        <div class="d-flex align-items-center mt-2">
+                            <div class="mr-2">
+                                <a
+                                    href=""
+                                    ng-click="$ctrl.scrollTo('engine')"
+                                    data-translate="pci_database_add_spec_engine"
+                                ></a>
+                                <span
+                                    ng-bind="$ctrl.model.engine.getLabel()"
+                                ></span>
+                            </div>
+                            <img
+                                ng-src="{{$ctrl.ENGINE_LOGOS[$ctrl.model.engine.name]}}"
+                                width="60px"
+                                height="40px"
+                                alt="engine logo"
```

Comment:
Consider to have to translate this.

Reply:
Added translation here: https://github.com/ovh/manager/pull/6905/commits/80db90fb5024122d9219c182593f7dce517f26e9